src/models/span.ts

```diff
@@ -0,0 +1,37 @@
+import { KeyValue, Log, Reference, SpanContext } from '../types';
+
+export class Span {
```

Comment:
missing `implements <Name Of Interface>`

Reply:
this could look like this:
```
export class Span implements types.Span{
  readonly spanContext: SpanContext;
  readonly operationName: string;
  readonly references: Reference[];
  readonly startTimestamp: Date;
  readonly durationMicros: string;
  readonly tags: KeyValue[];
  readonly logs: Log[];
  constructor(props: Span) {
    this.spanContext = props.spanContext;
    this.operationName = props.operationName;
    this.references = props.references;
    this.startTimestamp = props.startTimestamp;
    this.durationMicros = props.durationMicros;
    this.tags = props.tags;
    this.logs = props.logs;
  }
}
```